### DISCUSSION TITLE
bug/problem: close request removes tab in SB if window closing is not confirmed ("stay")

### DISCUSSION THREAD

**Discussion by Elementari** [mbnuqw/sidebery#1636] [General]
Some websites have a functionality that when closing a window they will ask if you want to actually close, esp. if you have entered information into a form on the website, in order to prevent loss of information.
At least on fastmail message composition, choosing "stay"  in firefox will cause the tab to disappear in sideberry, but not the tabs bar of FF. searching for the tab in sideberry with sideberry search feature will cause the tab to re-appear. I was able to reproduce the problem.

**Comment by albino1** [comment]
https://github.com/mbnuqw/sidebery/issues/1246 maybe?  Supposedly fixed, but I guess this is an additional problem.
